src/DependencyInjection/Compiler/CollectAsyncEventNames.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace SimpleBus\AsynchronousBundle\DependencyInjection\Compiler;
+
+use SimpleBus\SymfonyBridge\DependencyInjection\Compiler\CollectServices;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class CollectAsyncEventNames implements CompilerPassInterface
```

Comment:
Maybe it should be `CollectAsyncronousEvents`

It collects all events that are handled asynchronous.